packages/web-components/package.json

```diff
@@ -97,7 +97,7 @@
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
     "@microsoft/fast-element": "^1.0.0",
-    "@microsoft/fast-foundation": "1.16.0",
+    "@microsoft/fast-foundation": "1.16.1",
```

Comment:
@khamudom I think the issue here was that this was hard coded. We should have this opted-in to semver. What happens if you change this to `^1.16.0`?

Reply:
updated